test/types/tsconfig.json

```
@@ -0,0 +1,28 @@
+{
+	"compilerOptions": {
+		"target": "es2019" /* Specify ECMAScript target version: 'ES3' (default), 'ES5', 'ES2015', 'ES2016', 'ES2017', 'ES2018', 'ES2019' or 'ESNEXT'. */,
+		"lib": [
+			"es2019.object",
+		],
```

Why are we specially adding `es2019.object`? 🤔 That adds `Object.fromEntries` which I don't see that we are using anywhere...

Since we are supporting Node.js 10.x, maybe we should set this to:

```suggestion
		"target": "es2018",
		"lib": ["es2018"],
```

ref: https://stackoverflow.com/a/57607634/148072